include/libswiftnav/signal.h

```diff
@@ -37,16 +58,14 @@ enum band {
 
 typedef struct __attribute__((packed)) {
   u16 sat;
-  u8 band;
-  u8 constellation;
+  enum code code;
+  u8 reserved;
```

Comment:
What matters is the SBP representation. We should keep the extra byte there, but here it doesn't matter.